setup.cfg

```diff
@@ -30,12 +30,13 @@ packages = find_namespace:
 zip_safe = True
 include_package_data = True
 setup_requires =
-  setuptools_scm
+  setuptools >= 61.2.0
+  setuptools_scm >= 6.4.2
```

Comment:
`setuptools_scm  >= 6.4.2` does not match exactly with the version in `requirements.txt`, where it is `==6.4.2` instead.

Reply:
Done!